internal/algod/linux/linux.go

```
@@ -24,6 +26,20 @@ type Algod struct {
 	DataDirectoryPath string
 }
 
+func CheckConflictingUser() {
```

This should return the error to the caller

---

If the user does not exist (such as when there is no node installed at all) then you get an error when you look them up

I've omitted the error handling for other error types because it isn't a critical condition/check that we are making